website/server/models/task.js

```diff
@@ -243,7 +243,7 @@ export let TodoSchema = new Schema(_.defaults({
   dateCompleted: Date,
   // TODO we're getting parse errors, people have stored as "today" and "3/13". Need to run a migration & put this back to type: Date see http://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript
   date: String, // due date for todos
-}, dailyTodoSchema()), subDiscriminatorOptions);
+}, dailyTodoSchema(), habitDailySchema()), subDiscriminatorOptions);
```

Comment:
sorry, I got confused by the name. It'll add the fields in `habitDailySchema` but right now is only history. So it's fine to add them but the name of the object should be renamed or it's confusing